### PR TITLE
docs: update for 0.25 release

### PR DESCRIPTION
**Pull request by dougm**

### CODE REVIEW

File: govc/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
 # changelog
 
+### 0.25.0 (2021-04-16)
```
Comment (embano1) [inline govc/CHANGELOG.md:3]: the new version behavior (-l option) is missing

**Review by embano1 — CHANGES_REQUESTED** [review]
Do we also generate a changelog for vcsim?

File: govc/CHANGELOG.md (hunk 1 above)
Comment (dougm) [reply govc/CHANGELOG.md:3]: I'll add that.  There's other stuff missing too.  Need a better way that going back through git history for future releases.
I didn't bother with the top-level govmomi/CHANGELOG.md this time and have never maintained one for vcsim.

**Review by embano1 — APPROVED** [review]
LGTM

**Pull request merged**